src/AzureSqlAction.ts

```diff
@@ -123,7 +122,7 @@ export default class AzureSqlAction {
 
         switch (inputs.sqlpackageAction) {
             case SqlPackageAction.Publish: {
-                args += `/Action:Publish /TargetConnectionString:"${inputs.connectionString.connectionString}" /SourceFile:"${inputs.dacpacPackage}"`;
+                args += `/Action:Publish /TargetConnectionString:"${inputs.connectionConfig.ConnectionString}" /SourceFile:"${inputs.dacpacPackage}"`;
```

Comment:
Might you need to escape any doublequotes inside the connection string here?

Reply:
Good point, I'll add a test case for that also

Reply:
Actually I don't believe this needs to be escaped. This string is the same as the one user supplied on the command line, so it should be properly escaped already otherwise it will fail validation early on.